packages/backend/package.json

```diff
@@ -54,9 +54,9 @@
         "@opentelemetry/sdk-trace-node": "^1.13.0",
         "@opentelemetry/semantic-conventions": "^1.13.0",
         "@rudderstack/rudder-sdk-node": "^1.1.3",
-        "@sentry/node": "^7.114.0",
-        "@sentry/profiling-node": "^7.114.0",
-        "@sentry/types": "^7.114.0",
+        "@sentry/node": "8.7.0",
+        "@sentry/profiling-node": "8.7.0",
+        "@sentry/types": "8.7.0",
```

Comment:
https://github.com/getsentry/sentry-javascript/releases/tag/8.7.0

On https://github.com/getsentry/sentry-javascript/releases/tag/8.8.0 they included these changes, and then raised this:

```
If you are still encountering issues with OpenTelemetry instrumentation and ESM, please let us know.
```
Builds & Sentry work great with this version